api/http/management.go

```diff
@@ -90,7 +90,7 @@ func (h *ManagementHandler) CreateIntegration(c *gin.Context) {
 	//      - service
 	//      - registry read/write
 
-	err = h.app.CreateIntegration(ctx, integration)
+	inserted, err := h.app.CreateIntegration(ctx, integration)
```

Comment:
will the inserted integration here include the id?

Reply:
Yes, we generate it in the store service before inserting the doc.

Reply:
right; I've completely missed [this](https://github.com/mendersoftware/iot-manager/blob/b1ab79f456b26bf9d542e944db6f3d08c52573cf/store/mongo/datastore_mongo.go#L295) line